src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public class Main {
```

Comment:
The Main class should not contain an empty line at the start of its definition. Please remove the empty line to follow best practices.